tests/integration/test_password_rotation.py

```diff
@@ -77,3 +77,49 @@ async def test_password_rotation(ops_test: OpsTest):
         if not await unit.is_leader_from_status():
             await restart_patroni(ops_test, unit.name)
             assert await check_patroni(ops_test, unit.name, restart_time)
+
+
+async def test_password_from_secret_same_as_cli(ops_test: OpsTest):
```

Comment:
```suggestion
@pytest.mark.juju3
async def test_password_from_secret_same_as_cli(ops_test: OpsTest):
```
This test can't run on Juju 2.9